src/assets/browser/app/src/controller/scheduler/schedmain.rs

```diff
@@ -66,7 +66,7 @@ impl SchedulerMain {
     fn check_tempo(&mut self, burst: bool) {
         let now = browser_time();
         self.jank.detect(burst,now/1000.);
-        self.set_timesig(self.jank.gear());
+        //self.set_timesig(self.jank.gear());
```

Comment:
should this go away?

Reply:
This commented-out-code will go when the scheduler is reimplemented. At the moment this isn't clear, so I'll put a proper comment to that effect.

Reply:
D'oh! I should have read the comment to the commented line! :man_facepalming: